README.md

```diff
@@ -2,7 +2,47 @@
 
 Automation of OpenShift 3 using [Ansible](http://www.ansible.com/)
 
-## **Note: This section is currently under active development.**
+## Provisioning Quickstart
+
+### Local Setup (one time, only)
+
+In addition to _cloning this repo_, you'll need the following:
+
+* Access to an OpenStack environment using an link:http://docs.openstack.org/user-guide/common/cli-set-environment-variables-using-openstack-rc.html[OpenStack RC File]
+  * File should be placed at `~/.config/openstack/openrc.sh`
+* A link:https://github.com/naturalis/openstack-docs/wiki/Howto:-Creating-and-using-OpenStack-SSH-keypairs-on-Linux-and-OSX[Key-pair created in OpenStack]
+* Docker installed (`yum install -y docker` on RHEL/Centos, `dnf install docker -y` on Fedora)
+  * If you plan to run docker as yourself (non-root), your username must be added to the `docker` user group.
+* An `~/.ansible.cfg` file containing the following:
+```
+[defaults]
+roles_path= /root/repository/rhc-ose/rhc-ose-ansible/roles:/root/repository/openshift-ansible/roles
```

Comment:
Looks incorrect - since the repo is now `casl-ansible`, that's what the first path should be pointed to. Also, would be good to add a note in this area to say that if the default location of the clone is not used, this needs to be updated as well.